modules/virt-understanding-live-migration.adoc

```diff
@@ -1,20 +1,19 @@
 // Module included in the following assemblies:
 //
 // * virt/live_migration/virt-live-migration.adoc
+// * virt/upgrading-virt.adoc
 
 [id="virt-understanding-live-migration_{context}"]
 = Understanding live migration
```

Comment:
Just wondering if we need to include this module in the upgrade assembly. It breaks the flow of content IMO and you also link to it in the Additional resources section.

Reply:
You aren't the first to suggest this, so I will consider removing it. :)